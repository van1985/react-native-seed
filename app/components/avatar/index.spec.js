import React from 'react';
import renderer from 'react-test-renderer';
import Avatar from './index';

test('renders correctly', () => {
  const tree = renderer.create(<Avatar
    width={60}
    height={60}
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
