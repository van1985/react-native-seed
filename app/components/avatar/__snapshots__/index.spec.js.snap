// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "borderColor": "rgba(0,0,0,0.2)",
        "borderRadius": 30,
        "borderWidth": 1,
        "height": 60,
        "justifyContent": "center",
        "opacity": 1,
        "width": 60,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Image
      source={
        Object {
          "testUri": "../../../app/assets/silhouette200.png",
        }
      }
      style={
        Object {
          "borderRadius": 30,
          "height": 60,
          "width": 60,
        }
      }
    />
  </View>
</View>
`;
